qutebrowser/commands/runners.py

```diff
@@ -285,6 +286,14 @@ def run(self, text, count=None):
             else:
                 result.cmd.run(self._win_id, args)
 
+            mode_manager = objreg.get('mode-manager', scope='window',
+                                      window=self._win_id)
+            if (result.cmdline[0] != 'repeat-command' and
+                    result.cmdline[0] != 'prompt-accept'):
```

Comment:
I used to use a tuple for constant things, but nowadays I'd actually prefer a list as a tuple is something more heterogeneous (i.e. it makes sense to have a `(x, y)` point as a tuple, but a list of points would be a list).

At some point I should probably change it to a list everywhere :wink: